pages/outreachy.md

```diff
@@ -55,6 +55,28 @@ the mailing lists or Gitter channels.
 
 Interested in becoming a mentor? Contact one of the coordinators!
 
+## Contribute
+
+As part of the application process, the Outreachy program recommends the candidates to make 
```

Comment:
"recommends that candidates make"

Reply:
Only other comment is whether to use the new `good first issue` label that github is now advertising, which would be relevant after outreachy has finished?

Reply:
LGTM and will merge once this edit is in.  👍

Reply:
About the `good first issue`, I can certainly add that to the issues that have been tagged with `outreachy` already. Not sure if they will always be a 1 to 1, so, can we keep both for now?

Reply:
> recommends that candidates make

Done!

Reply:
+1